src/Engine/UnitActionType.java

```diff
@@ -17,20 +17,24 @@ public GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor)
   }
   public abstract GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor, boolean ignoreResident);
   public abstract String name();
+  public boolean shouldConfirm = false;
```

Comment:
Public?

Reply:
The plan was vaguely to pull the "Is this targeted" dealie out of `GameActionSet`, but I didn't end up doing that.

Ideally also it'd be `final`, but that would mean copying it into every constructor, heh.

Reply:
This works well enough for now, anyway.